website_legal_page/views/reusable_templates.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <!-- Base templates for submodules -->
+    <template id="acceptance_full">
+        I accept the <a href="/legal/advice">legal advice</a>, the
```

Comment:
@eLBati sorry, don't know how but I lost my comment here... same story: can you wrap it (maybe in a <span />) w/ proper CSS class?

Reply:
Done